src/app.js

```diff
@@ -9,4 +9,6 @@ app.listen(3000, function() {
      console.log('Example app listening on port 3000!')
 })
 
-
+app.use(express.static('/public'))
+app.use(express.static('/files'))
+app.use(express.static(__dirname, { index: 'index.html' }));
```

Comment:
You can replace these three lines with one:

```js
app.use(express.static(path(__dirname, 'public')));
```

You'll also need to require the 'path' module at the top, like this:

```js
const path = require('path');
```

Path is a built-in Node module (no `npm install` needed) that comcatenates strings into an absolute directory path, and `__dirname` is a dynamic constant that always resolves to the directory of the current file.